src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <a href="https://www.moyo.ua/monoblok_apple_a1419_imac_27_retina_5k_mned2ua_a/361498.html?drsq=%D0%9C%D0%BE%D0%BD%D0%BE%D0%B1%D0%BB%D0%BE%D0%BA+APPLE++A1419+iMac+27%22+Retina+5K+%28MNED2UA%2FA%29">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac"
+          >
```

Comment:
```suggestion
        <img
          class="card__image"
          src="images/imac.jpeg"
          alt="iMac"
        >
```
закрывающая скобка должна быть на одной линии с открывающей